src/ensembl/src/shared/radio/Radio.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react';
+
+import defaultStyles from './Radio.scss';
+
+export type radioOption = {
```

Comment:
Here and below: can we name types starting with capital letters?

Reply:
That's done.